### DISCUSSION TITLE
MS Learn Cosmos DB

### DISCUSSION THREAD

**Discussion by mumby0168** [IEvangelist/azure-cosmos-dotnet-repository#102] [Show and tell]
There has been some really good MS learn modules added for Cosmos DB around modelling and partitioning data.

Thought they where worth sharing see below.

1. [Model and partition your data in Azure Cosmos DB](https://docs.microsoft.com/en-us/learn/modules/model-partition-data-azure-cosmos-db/)
2. [Optimize databases by using advanced modeling patterns for Azure Cosmos DB](https://docs.microsoft.com/en-us/learn/modules/advanced-modeling-patterns-azure-cosmos-db/)

**Comment by IEvangelist** [comment]
Love these, thank you @mumby0168.
